src/redux/books/bookSlice.js

```diff
@@ -1,40 +1,83 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
- [ ] Please Consider removing this and all other disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase.